frontend_tests/node_tests/compose.js

```diff
@@ -293,6 +294,18 @@ test_ui("send_message", ({override, override_rewire, mock_template}) => {
 
 test_ui("enter_with_preview_open", ({override, override_rewire}) => {
     mock_banners();
+    mock_stream_header_colorblock();
+    compose.update_on_recipient_change = noop;
+    let stream_value = "";
+    compose_ui.compose_stream_widget = {
```

Comment:
Needing to do this repeatedly in tests feels a bit messy, not sure if there's a better way to handle this.